### ISSUE TITLE
Current formatting makes code less readable in some scenario

### ISSUE THREAD

**Issue by 1toldyou** [robototes/RapidReact2022#44]
The threshold of long-string is be to low, breaks line in unwanted position
for example I have a comment
```
/*
 * so as mentioned above this subsystem is only for hardware on top of existing hardware
 */
```
and forcefully change into the following
```
/*
 * so as mentioned above this subsystem is only for hardware on top of existing
 * hardware
 */
```
quite unnecessary for the sentence that's two or three words over the threshold
although I could break the sentence in the middle to make it reads smooth even in multiple lines, but a modern 16:9 screen should be able to display a line with 100 characters
Could the formatter be little relax/flexible on the long-string detection?

**Comment by obj-obj** [comment]
@1toldyou there's probably an option for this in the `eclipse-formatter.xml`

**Comment by obj-obj** [comment]
Yep, it's `org.eclipse.jdt.core.formatter.comment.line_length` which is currently set to `80`.
What do you think a good value would be?

**Comment by obj-obj** [comment]
Fixed with #46
